src/hotspot/os/linux/os_linux.cpp

```diff
@@ -5367,19 +5367,22 @@ bool os::start_debugging(char *buf, int buflen) {
 //    |                        |/
 // P2 +------------------------+ Thread::stack_base()
 //
-// ** P1 (aka bottom) and size (P2 = P1 - size) are the address and stack size
+// ** P1 (aka bottom) and size are the address and stack size
 //    returned from pthread_attr_getstack().
+// ** P2 (aka stack top or base) = P1 - size)
```

Comment:
Preexisting, but isn't P2 = P1 + size? Also an extra parenthesis was left at the end.

Reply:
Good catch!